af_set_enable_stacktrace (af/util.h) does not link properly, stacktrace machanism hangs.

on the latest release, using this function causes linker error (Windows VS2019):

`error LNK2019: unresolved external symbol __imp_af_set_enable_stacktrace`

I examined the source code, but could not tell what went bad in this particular case.

the reason I came to this function is because boost-stacktrace mechanism causes hangs / deadlocks on Windows. I get this hang when an exception is thrown and the boost-stacktrace calls [IDebugSymbols3::getNameByOffset](https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/dbgeng/nf-dbgeng-idebugsymbols3-getnamebyoffset).

@9prady9 did you manage to reproduce this? I have a standalone console app for you.

this is the code:

```
#include <iostream>
#include <arrayfire.h>
#include <af/util.h>
#pragma comment(lib, "afopencl.lib")
int main()
{
    af_set_enable_stacktrace(0);
    std::cout << "Hello World!\n";
}

```
arrayfirebug.obj : error LNK2019: unresolved external symbol __imp_af_set_enable_stacktrace referenced in function main

OKay, I have a console app created with this code and set the include directories in the Project's `Additional Include Directories` property. I have the `LINK : fatal error LNK1104: cannot open file 'afopencl.lib'` as I expected since it doesn't have path. After I changed the pragma to the following, I got similar error.
```
#pragma comment(lib, "<...>\\ArrayFire\\v3\\lib\\afopencl.lib")
```
I am curious as to where you library search path is coming from. Are you setting that `Additional Library Directories` in properties instead of giving path in the code directly ?

One definite difference is that Console App was created by default with x86 target which isn't compatible with our website binaries that are generated for x64 target. Please check that isn't the case in your test project too.

Having said that, I am able to reproduce the issue against x64 target. I am looking into it.

--- a/src/api/c/error.cpp
+++ b/src/api/c/error.cpp
@@ -10,6 +10,7 @@
 #include <common/err_common.hpp>
 #include <af/device.h>
 #include <af/exception.h>
+#include <af/util.h>
 
 #include <algorithm>
 #include <string>

--- a/src/api/unified/error.cpp
+++ b/src/api/unified/error.cpp
@@ -10,6 +10,7 @@
 #include <af/array.h>
 #include <af/device.h>
 #include <af/exception.h>
+#include <af/util.h>
 #include <algorithm>
 #include "symbol_manager.hpp"
 
@@ -41,4 +42,8 @@ void af_get_last_error(char **str, dim_t *len) {
         auto func = reinterpret_cast<af_func>(LOAD_SYMBOL());
         func(str, len);
     }
+}
+
+af_err af_set_enable_stacktrace(int is_enabled) {
+    CALL(af_set_enable_stacktrace, is_enabled);
 }

